_bmad-output/implementation-artifacts/stories/3-3-safe-subscriber-unregistration.md

```
@@ -0,0 +1,300 @@
+**Oracle implementation** (`internal/adapter/storage/oracle/subscriptions.go`):
+- Mirror `RegisterNewSubscriber` pattern: `BEGIN OMNI_TRACER_API.Unregister_Subscriber(:subscriberName); END;`
+- Use `subscriber.ConsumerName()` as the subscriber name (FunnyName)
+
+**Stub implementations to update** (add `return nil` no-ops):
+- `internal/service/tracer/tracer_service_test.go` → `stubDatabaseRepository`
+- `internal/service/subscribers/procedure_generator_test.go` → `stubDBRepo`
+- `internal/adapter/ui/database_settings_test.go` → `MockDatabaseRepository`
+
+### File Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)